lib/rollbar/item.rb

```diff
@@ -101,16 +101,24 @@ def dump
       # from an async handler job, which can be serialized.
       stringified_payload = Util::Hash.deep_stringify_keys(payload)
       result = Truncation.truncate(stringified_payload)
+
       return result unless Truncation.truncate?(result)
 
-      original_size = Rollbar::JSON.dump(payload).bytesize
-      final_size = result.bytesize
-      notifier.send_failsafe("Could not send payload due to it being too large after truncating attempts. Original size: #{original_size} Final size: #{final_size}", nil)
-      logger.error("[Rollbar] Payload too large to be sent: #{Rollbar::JSON.dump(payload)}")
+      handle_too_large_payload(stringified_payload, result)
 
       nil
     end
 
+    def handle_too_large_payload(stringified_payload, final_payload)
+      original_size = Rollbar::JSON.dump(stringified_payload).bytesize
+      final_size = final_payload.bytesize
+      uuid = stringified_payload['data']['uuid']
+      host = stringified_payload['data']['server']['host']
```

Comment:
If 'server' or 'host' are missing, will this cause an error? If so, we should wrap it in a begin-rescue. Here's how pyrollbar does it: https://github.com/rollbar/pyrollbar/blob/master/rollbar/__init__.py#L1252-L1259